### DISCUSSION TITLE
Capacitor Share Plugin ios issues

### DISCUSSION THREAD

**Discussion by sbodla** [ionic-team/capacitor#3955] [Q&A]
Hi,
I am using Capacitor SharePlugin for sharing some text, url to social apps. I have issues with some apps like Facebook and Slack where i cannot see 'text' while sharing on Facebook/Slack. I see only url value on Fb post/slack message. This is only issue in ios ( ios>13). It used to work before in older ios versions as well. Anyone facing same issue or know of issue? Is this something related to specific ios version/plugin need to be updated?

Capacitor Core Version - 2.4.0

Sample code:
import { Plugins } from '@capacitor/core';
const { Share } = Plugins;

Version 1:
async shareToPublic() {
    await Share.share({
      text:  "Sample Text Here",
      url: 'https://www.google.com',
      dialogTitle: 'Share with',
    });
  }

Version 2:
async shareToSocial() {
await Share.share({
      text: 'Please share our app',,
      title: 'Here We go:'
      url: 'https://www.github.com',
      dialogTitle: 'Share with',
    })
}

----
Both the versions just show url(with preview) in the Fb post/Slack message. I see text in pop up window while sharing in facebook though. I am not sure why that text is not being passed to Facebook post/slack message.

 But, other apps like Whatsapp/text message shares everything( text, url with preview). Any help is appreciated.

**Comment by jonahtwalker** [comment]
@sbodla 

I've been digging into this trying to understand why I've got the same result - I'd like to set the `message` for a few Apps, including FB Messenger and Signal.

In the SocialSharing plugin repo, they include some disclaimers in _their_ documentation: https://github.com/EddyVerbruggen/SocialSharing-PhoneGap-Plugin#4-usage-on-ios-and-android

For the sake of saving you a FB login, this is the damning post:

![Screen Shot 2021-01-05 at 16 27 46](https://user-images.githubusercontent.com/64275677/103714932-19a55b80-4f74-11eb-8048-2d5727c0cb38.png)

Haven't found confirmation like that for Signal, but I'm still hunting for a workaround or a more recent final word on the issues. I will update here if/when we get a solution!
